api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -510,38 +519,24 @@ public async Task<IActionResult> JoinClipsAsync([FromRoute] int? locationId, [Fr
     /// </summary>
     /// <param name="updatedBefore">optional, only upload files updated before the specified date</param>
     /// <param name="limit">optional, only upload limit files</param>
+    /// <param name="force">optional, force upload files</param>
     /// <returns>uploaded files and failed uploads</returns>
     [HttpPost("upload-files-to-s3")]
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(Dictionary<string, List<string>>), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Storage" })]
-    public async Task<IActionResult> UploadFilesToS3([FromQuery] DateTime? updatedBefore = null, [FromQuery] int? limit = null)
+    public async Task<IActionResult> UploadFileToS3Async([FromQuery] DateTime? updatedBefore = null, [FromQuery] int? limit = null, [FromQuery] bool force = false)
```

Comment:
When a file was updated last has little meaning or value to MMI.  We use those for audit fields, but generally nothing else.  Uploading to S3 probably should be based on the `Content.PublishedOn` date, as that has a relationship to the license.  Also there will be a lot of historical content that was recently updated, but it's old and should be moved to S3.

Reply:
Wrong word, it's a filter base on `PublishedOn` ,  I will change to a proper one like `PublishBefore`.